002-es-modules.md

```diff
@@ -0,0 +1,540 @@
+| Title  | ESM Interoperability |
+|--------|-----------------------------|
+| Author | @bmeck                      |
+| Status | DRAFT                       |
+| Date   | 2016-01-07                  |
```

Comment:
hm. I never defined that in the README. but sure. time of last update sounds good.